### DISCUSSION TITLE
Identificar firma de certificado: publico (FNMT), libre (letsencrypt) o Privado

### DISCUSSION THREAD

**Discussion by ianvatega** [PucelaBits/websegura#134] [Ideas]
Indicar el tipo de certificado.

Si utilizan certificados SSL libres como es el caso de Let’s Encrypt o de publicas como fnmt o si son compradas a empreses privadas. 

En cualquier caso indicar el coste y comparlo con la libre y/o publica.

En el primer caso aunque sea gratis, aportar X dinero al proyecto al año por tal de ayudar al mantenimiento y mejora de los mismos. ( Se ahorraría mucho que los SSL de pago, a parte ayudan a su difusión para otro tipos de webs para empresas, entidades, personales etc) 

Todo esto se podría asociar como ejemplo de la campaña de FSFE 👉🏻 publiccode.eu/es/

También creo que son aplicables algunos de los 17 objetivos de ODS2030. Creo que tocaría algunas metas de estos puntos.

Objetivo 9. Industria, innovación e infraestructura
Objetivo 16. Paz, justicia e instituciones sólidas
Objetivo 17. Alianzas para lograr los objetivos

**Comment by ianvatega** [comment]
https://www.cert.fnmt.es/catalogo-de-servicios/certificados-electronicos/certificado-servidor

https://letsencrypt.org/es/how-it-works/

**Comment by nukeador** [comment]
¿Qué señalaría que usara o u otro? Mientras sea de una CA reconocida daría igual para el tema de seguridad ¿no?

**Reply by ianvatega** [reply]
Por tema de transparencia y de recursos públicos, también que es mejor tener una CA pública y/o libre que una empresa privada. 

A parte muchas instituciones sobretodo pequeñas creo que les puede ser más fácil aplicar un certificado let's encryt que uno de pago que quizás tenga que pedir presupuestos etc... y se demore más.( o eso creo(

**Reply by nukeador** [reply]
¿Por qué crees que es mejor tener una CA pública y/o libre que una empresa privada? ¿Hay algún estudio al respecto?

En el tema de presupuesto, dado que lets encrypt es gratis, el precio de los certificados de otras empresas ha bajado mucho (menos de 40€ al año), no creo que signifique nada para cualquier entidad pública.

**Reply by ianvatega** [reply]
Pero para el ahorro sip.

**Comment by ianvatega** [comment]
[https://datos.gob.es/es/catalogo/e05068901-lista-de-confianza-de-prestadores-cualificados-de-servicios-electronicos-de-confianza-tsl1](https://datos.gob.es/es/catalogo/e05068901-prestadores-de-servicios-electronicos-de-confianza)

[https://datos.gob.es/es/catalogo/e05068901-prestadores-de-servicios-electronicos-de-confianza](https://datos.gob.es/es/catalogo/e05068901-prestadores-de-servicios-electronicos-de-confianza)
